jobserver/settings.py

```diff
@@ -349,7 +349,10 @@
 # interactive functionality
 LOCAL_GIT_REPOS = BASE_DIR / "repos"
 
-# Releases storage location.
+# Released files per-file size limit
+RELEASE_FILE_SIZE_LIMIT = env.int("RELEASE_FILE_SIZE_LIMIT", default=16777216)  # 16Mb
```

Comment:
```suggestion
RELEASE_FILE_SIZE_LIMIT = env.int("RELEASE_FILE_SIZE_LIMIT", default=16*1024*1024)  # 16Mb
```

Encoding it like this makes it easy to adjust later?

Reply:
ooh yeah, fewer magic numbers 👍